src/client/pages/OfferNew/index.tsx

```diff
@@ -72,9 +73,10 @@ export const OfferNew: React.FC = () => {
   }
 
   if (!loadingQuoteBundle && !data?.quoteBundle) {
-    throw new Error(
+    captureSentryError(
       `No quote returned to show offer with (quoteIds=${quoteIds}).`,
```

Comment:
This message is a bit strange? Maybe we could write something like 
```suggestion
    captureSentryError(
      `No quote data could be returned for quoteIds: ${quoteIds}.`,
```

Also, could we be helped if we got the `error` from the query on line 52 and logged that to Sentry as well?

Reply:
@styrelius the error I got is quite strange - something like "Currency Code may not be null". Currently, we log all GraphQL request errors to Sentry so we will see it there.

Reply:
Oh, yes of course 🧠